src/app/participant-page/participant-page.component.html

```diff
@@ -915,51 +915,62 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
                 </app-medical-record-abstraction>
               </div>
             </tab>
-<!--            tabs from fieldSettings-->
-            <ng-container *ngFor="let tab of settings['null']">
-              <ng-container *ngIf="displayTab(tab)">
-                <tab heading="{{getDisplayName(tab.columnDisplay, tab.columnName)}}" [active]="tabActive(tab.columnDisplay)" (select)="onSelect($event, tab.columnDisplay)" >
-<!--                  <br/>-->
-
-<!--                  <div class="Float&#45;&#45;left Width&#45;&#45;100">-->
-<!--                    <b class="Group&#45;&#45;Title">Jump to:</b>-->
-<!--                  </div>-->
-<!--                  <ng-container *ngFor="let setting of settings[tab.columnName]; let settingIndex = index">-->
-<!--                    <div class="Float&#45;&#45;left" *ngIf="setting.displayType === 'GROUP'">-->
-<!--                      <a class="Abstraction&#45;&#45;Jump&#45;&#45;Group" pageScroll href="{{setting.columnName}}">{{setting.columnDisplay}}</a>-->
-<!--                    </div>-->
-<!--                    <ng-container *ngIf="settingIndex < settings[tab.columnName].length - 1">-->
-<!--                      <div class="Float&#45;&#45;left Vertical&#45;&#45;line"></div>-->
-<!--                    </ng-container>-->
-<!--                  </ng-container>-->
-<!--                  <div class="Float&#45;&#45;left Width&#45;&#45;100">-->
-<!--                    <hr style="border-color: black">-->
-<!--                  </div>-->
-
-                  <ng-container *ngFor="let setting of settings[tab.columnName]; let settingIndex = index">
-                    <div class="Float--left Width--100">
-                      <div *ngIf="setting.displayType === 'GROUP'">
-                        <fieldset style="border: 1px black solid">
-                          <legend class="Group--Title" id="{{setting.columnName}}">{{setting.columnDisplay}}</legend>
-                          <ng-container *ngFor="let field of settings[setting.columnName]; let fieldIndex = index">
-                            <app-form-data [fieldSetting]="field" [participantData]="getParticipantData(field, tab.columnName)"
-                                           [activityData]="getActivityData(field)" [activityOptions]="getActivityOptions(field)"
-                                           (patchData)="formPatch($event, field, setting)"></app-form-data>
-                            <hr *ngIf="fieldIndex !== settings[setting.columnName].length-1" style="border-color: lightgray">
+
+            <!-- <ng-container *ngFor="let relative of participant.participantData; trackBy: trackById;">
+              <div>
+                {{relative.dataId}}
+              </div>
+            </ng-container> -->
+            
```

Comment:
can that get removed?

Reply:
Yes, forgot to delete it, was just testing something